packages/help-center/src/components/help-center-chat-option.scss

```diff
@@ -0,0 +1,102 @@
+@import "@automattic/typography/styles/variables";
+
+.help-center-contact-support {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	gap: 16px;
+
+	& > button {
```

Comment:
Fixed disabled state styling. It was previously not applied correctly for the email contact support button.